src/api/config/block-sync-params.md

```diff
@@ -0,0 +1,27 @@
+# Block Sync Parameters
+
+Explain module
```

Comment:
Honestly, I haven't heard about that part before, so any explanation would be helpful. I wonder who can help?